.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Installation issue
+about: Report issues with the installation process of Manim
+title: ""
+labels: bug, installation
+assignees: ''
+
+---
+
+#### Preliminaries
+
+- [ ] I have followed the latest version of the
+      [installation instructions](https://docs.manim.community/en/latest/installation.html).
+- [ ] I have checked the [troubleshooting problem]() and my problem is either not mentioned there,
+      or the solution given there does not help.
```

Comment:
```suggestion
- [ ] I have checked the [troubleshooting page](https://docs.manim.community/en/latest/installation/troubleshooting.html) and my problem is either not mentioned there,
      or the solution given there does not help.
```

Reply:
Well, seems my thoughts were somewhere else when I wrote that. Thanks for fixing it!